pkg/appliance/checks.go

```diff
@@ -15,11 +15,22 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func PrintDiskSpaceWarningMessage(out io.Writer, stats []openapi.StatsAppliancesListAllOfData) {
+func PrintDiskSpaceWarningMessage(out io.Writer, stats []openapi.StatsAppliancesListAllOfData, apiVersion int) {
```

Comment:
:pick: same here, you could remove apiVersion check here and only check if we have DiskInfo, otherwise fallback to DiskUsage

Reply:
The table header is a problem here though. Appliance stats are in a slice and is looped to add lines to the table. The table header though is set before looping the appliance stats slice, so `GetDiskInfoOk` is not available at that point, and removing the `apiVersion` param would make it even more complicated.

Reply:
could be solved by simple sharing the title between the different values, for example `Disk` or `Disk usage` and the user can deduce what it is based on the unit (percentage or megabyte).

Reply:
I'm pretty sure we are going to get confused users if we do that though. I'd rather have the code a little bit more complex than risk the confusion.